cinema/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import Movie
+
+class MovieSerializer(serializers.ModelSerializer):
+    title = serializers.CharField(required=True, max_length=255)
+    description = serializers.CharField(required=False)

```

Comment:
The `description` field in the serializer should have a `max_length` constraint of 500 to match the model definition. Consider adding `max_length=500` to ensure consistency.